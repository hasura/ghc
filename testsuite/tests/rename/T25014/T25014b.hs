-- Should not compile trying to export a missing name
module T25014b (A (foo)) where

data A
