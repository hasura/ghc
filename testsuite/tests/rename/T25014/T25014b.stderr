T25014b.hs:2:17: [GHC-76037]
     Not in scope: ‘foo’
     In the export: A(foo)